config/core/test-configs.yaml

```diff
@@ -2318,6 +2318,20 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
This is conflicting with an earlier PR which introduced the `smc` test plan.  Please rebase to resolve the conflict.

Reply:
Fixed